attestation-service/docs/grpc-as.md

```diff
@@ -14,16 +14,16 @@ Here are the steps of building and running gRPC Attestation Service:
 
 Build and install binary
 ```shell
-git clone https://github.com/confidential-containers/kbs
-cd kbs/attestation-service
+git clone https://github.com/confidential-containers/trustee
+cd trustee/attestation-service
 WORKDIR=$(pwd)
 make && make install
 ```
 
 Build and run container image
 ```shell
-git clone https://github.com/confidential-containers/kbs
-cd kbs
+git clone https://github.com/confidential-containers/trustee
+cd trustee/attestation-service
```

Comment:
should this be `cd trustee`? In the next line we specify `attestation-service` as part of the path to the Dockerfile

Reply:
Oh yes. Thanks for pointing out